### PR TITLE
testing/20190503_Timing

### PR DESCRIPTION
**Pull request by NicolaiFransen**
Crazy nice stuff!¡!¿¿¿

### CODE REVIEW

File: Software/Development/SemesterProject/App/analogAcquisitionManager.c [hunk 1]
```diff
@@ -456,9 +457,10 @@ void configureADCRegisters(void)
     EPwm1Regs.ETSEL.bit.INTEN = 1;                // Enable INT
     EPwm1Regs.ETSEL.bit.INTSEL = ET_CTR_ZERO;     // Select INT on Zero event
 
-    EPwm1Regs.ETSEL.bit.SOCAEN  = 1;        // Enable SOC on A group
-    EPwm1Regs.ETSEL.bit.SOCASEL = 1;        // Select SOC from CMPA on upcount
-    EPwm1Regs.ETPS.bit.SOCAPRD  = 1;        // Generate pulse on 1st event
+    EPwm1Regs.ETSEL.bit.SOCAEN  = 1;            // Enable SOC on A group
+    EPwm1Regs.ETSEL.bit.SOCASEL = ET_CTR_ZERO;  // To trigger interrupt at PWM high
+//    EPwm1Regs.ETSEL.bit.SOCASEL = ET_CTR_PRD;   // To trigger interrupt at PWM low
```
Comment (murgui) [inline Software/Development/SemesterProject/App/analogAcquisitionManager.c:462]: Nice

File: Software/Development/SemesterProject/System/scheduler.c [hunk 2]
```diff
@@ -23,10 +23,10 @@
 static enum taskListTag
 {
     task5msItem,
-    task10msItem,
     task20msItem,
     task200msItem,
     task1sItem,
+    task10msItem,
```
Comment (AitorTeran) [inline Software/Development/SemesterProject/System/scheduler.c:29]: Why did you change the position from the right place to the wrong place? C'mon Nicolai...

File: Software/Development/SemesterProject/System/scheduler.c [hunk 3]
```diff
@@ -201,10 +201,10 @@ Uint32 getSystemClock(void)
...
 //    Print stuff with UART here�!
-//    UARTIntPrint("Vc ", (int)getControlsupplyMeasurement() * 100);
-//    UARTIntPrint("DC ", (int)(getDCLinkMeasurement() * 100));
-//    UARTIntPrint("T ", (int)getThermometer1Measurement());
-//    UARTIntPrint("RPM ", (int)readRotorRPM());
+    //UARTIntPrint("Vc ", (int)getControlsupplyMeasurement() * 100);
+    UARTIntPrint("DC ", (int)(getDCLinkMeasurement() * 100));
+    UARTIntPrint("T ", (int)getThermometer1Measurement());
+    //UARTIntPrint("RPM ", (int)readRotorRPM());
 }
```
Comment (AitorTeran) [inline Software/Development/SemesterProject/System/scheduler.c:208]: We shouldnt merge into developp with things uncommented here right?

**Review by AitorTeran — APPROVED** [review]
Just minor things

File: Software/Development/SemesterProject/System/scheduler.c [hunk 4]
```diff
@@ -201,10 +201,10 @@ Uint32 getSystemClock(void)
 void printUART(void)
 {
 //    Print stuff with UART here�!
-//    UARTIntPrint("Vc ", (int)getControlsupplyMeasurement() * 100);
-//    UARTIntPrint("DC ", (int)(getDCLinkMeasurement() * 100));
-//    UARTIntPrint("T ", (int)getThermometer1Measurement());
-//    UARTIntPrint("RPM ", (int)readRotorRPM());
+    //UARTIntPrint("Vc ", (int)getControlsupplyMeasurement() * 100);
+    UARTIntPrint("DC ", (int)(getDCLinkMeasurement() * 100));
```
Comment (murgui) [inline Software/Development/SemesterProject/System/scheduler.c]: Printing by default?

**Review by murgui — APPROVED** [review]
Nice...ish¿¿
